src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -435,6 +427,19 @@ public static IEnumerable<string> GetInvalidStatusLine()
                 yield return "HTTP/1.12 200 OK";
                 yield return "HTTP/12.1 200 OK";
             }
+
+            // Skip these test cases on CurlHandler since the behavior is different.
+            if (PlatformDetection.IsWindows)
```

Comment:
I don't realize any general Unix PlatformDetection (there is specific detection, like OSX), so I take the reverse, which is only run on Windows.

Reply:
>I don't realize any general Unix PlatformDetection (there is specific detection, like OSX)

There are other classes/properties that can be used for Unix detection (which includes Mac OSX and Linux).

Reply:
The `[PlatformSpecific(TestPlatforms.AnyUnix)]` can only be applied to test method, not for static test data.